addons/web/static/tests/views/basic_model_tests.js

```diff
@@ -65,6 +65,29 @@ odoo.define('web.basic_model_tests', function (require) {
     }, function () {
         QUnit.module('BasicModel');
 
+        QUnit.test('context is given when using a resequence', async function (assert) {
+            assert.expect(1);
+
+            this.params.context = { active_field: 2 };
+            var model = createModel({
+                Model: BasicModel,
+                data: this.data,
+                mockRPC: function (route, args) {
+                    if (route === '/web/dataset/resequence') {
+                        assert.deepEqual(args.context, { active_field: 2 },
```

Comment:
could you check the read as well?

Reply:
It is done and the test is successful only with the fix :)